pyrometheus/__init__.py

```diff
@@ -342,37 +566,119 @@ def get_temperature(self, H_or_E, T_guess, Y, do_energy=False):
 
         num_iter = 500
         tol = 1.0e-6
-        T_i = T_guess
-        dT = 1.0
-        F  = H_or_E
-        J  = 0.0
-
-        for iter in range( 0, num_iter ):
-            F    -= he_fun( T_i, Y )
-            J    -= pv_fun( T_i, Y )
-            dT    = - F / J
-            T_i  += dT
-            if np.abs( dT ) < tol:
-                T = T_i
+        ones = _pyro_zeros_like(enthalpy_or_energy) + 1.0
+        t_i = t_guess * ones
```

Comment:
What does this do?

Reply:
I intended to allow the call:
`get_temperature(.., tguess=300.0, ...)`,
where tguess is a single scalar value.